### DISCUSSION TITLE
Jeet não está sendo mais mantido

### DISCUSSION THREAD

**Discussion by andersoonweb** [frontendbr/forum#134] [Perguntas e Respostas]
Pessoal, acabei de entrar no site do Jeet e achei essa mensagem: 

> Jeet is currently unmaintained. You are free to continue using it as-is, or to volunteer to take over maintenance yourself!

Os demais colegas estão utilizando o mesmo? 
Poderiam recomendar grids semelhante?

**Comment by willianjusten** [comment]
De fato o Jeet morreu e você pode ver o comentário de um dos mantenedores [aqui](https://github.com/mojotech/jeet/issues/468#issuecomment-209064606).

Muito disso se deve ao fato de que o cory simmons que é criador do Jeet ter criado um novo grid chamado [lost-grid](http://peterramsing.github.io/lost/), que nada mais é que o Jeet modificado para suportar Post-CSS, CSS puro dentre outros.

Eu confesso que ainda uso o Jeet em muitos projetos, não tenho uma real necessidade de mudar agora, mas para projetos iniciais, de fato, o Lost pode ser a melhor escolha agora.

**Comment by felipefialho** [comment]
Então, comecei a brincar com o Lost no nosso [projeto de eventos](https://github.com/frontendbr/eventos/tree/interface) e decidi migrar de vez em todos os novos projetos.

Ele funciona igual o Jeet mas trás novas features (como facilitador para usar Flexbox), e está com uma comunidade mais ativa agora.

Como ponto negativo, eu acho os exemplos de uso do Lost muito mal desenvolvidos na documentação. Tem milhares de possibilidades, mas mal são abordadas.

**Comment by danilovaz** [comment]
Eu não cheguei a usar o Jeet. Mas depois de ver várias pessoas comentando sobre esses grids na comunidade, resolvi embarcar no Lost e atualmente me encontro em um relacionamento sério com o mesmo :joy: :joy: :joy: 

Mas é como o @LFeh falou: a documentação por enquanto deixa muito a desejar.

**Comment by willianjusten** [comment]
A documentação do Lost é HORRÍVEL!!!

**Comment by felipefialho** [comment]
Acho muito estranho, porque sendo do mesmo criador do Jeet (que tem uma documentação legal), qual a lógica?

**Comment by willianjusten** [comment]
Muito provavelmente é pelo tempo...o cory simmons meio que se "desligou" do opensource por um tempinho....então a parada tá só sendo mantida por outros, que não estão muito preocupados... Mas a gente poderia ver de gerar a documentação hein. Já pensou, seria uma bela de uma contribuição.

Se bem que...deixa para lá, num to com vida para isso =x

**Comment by vitor-mendrone-deel** [comment]
Vejo isso como um problema. 

O Jeet era simples, não tinha nenhum segredo ou passo complexo pra começar a usar. Muitas vezes eu consegui ensinar pessoas a usá-lo em poucos minutos 

Isso não ocorre com o Lost, apesar de ainda não ser muito complexo, a documentação peca e ele cresceu em complexidade. A primeira vez que usei fiquei realmente Lost (<vozaguda>Piada bosta</vozaguda>) e ainda não encontrei ninguém que consiga explicá-lo em menos de 5 minutos. 

Vou usar o Lost em um projeto que recebi hoje mas ainda vejo o Jeet como um GS melhor por ser muito mais simples.

**Comment by danilovaz** [comment]
Então caras, eu não tô muito ligado no Jeet. Mas no Lost pelo que eu vi (posso estar enganado), parece que quem tomou a frente é o Peter Ramsing, não? E só de olhar as issues dá pra ver que o cara tá meio Lost (hahaha @Mendrone ) com tanta coisa a se fazer e ainda tem a documentação.

**Comment by danilovaz** [comment]
[backlog](https://waffle.io/peterramsing/lost)

**Comment by vitor-mendrone-deel** [comment]
Eu queria **MUITO** ter tempo pra fazer me voluntariar a manter o Jeet. Espero que alguma alma se ofereça pra isso e entenda que a magia da parada é a baixa complexidade. 

(Não adianta aparecer um mantenedor se o cara tentar transformar o Jeet no Lost. 

O Cory Simmons anda meio sumido dos trampos open source mesmo. Em uma issue aberta no repo do Jeet ele dizia que se a MojoTech liberasse os direitos do Jeet pra ele, ele continuaria mantendo a parada e daria créditos a eles pelo site.

**Comment by felipefialho** [comment]
Então, a maior vantagem do Lost na minha opinião é a facilidade de usar Flexbox se quiser, fica muito prático. A sacada de usar calc() para fazer os grids também é inteligente, pois se adapta em qualquer sistema.

Eu já pensei em contribuir para a documentação, mas não tenho experiencias em fazer documentações legais. 

@Mendrone Também achei o Lost um pouco confuso no inicio, mas vai por mim é totalmente culpa da documentação (e também não tem tanta issue pela internet afora ainda). É bem simples de usar e é praticamente igual o Jeet.

Porém algumas coisas mais complexas como o Grid Waffle eu não entendi até hoje por causa da documentação porca e falta de exemplos.

**Comment by andersoonweb** [comment]
Que pena, como o @Mendrone falou o jeet é bem simples e muito intuitivo. 

Vou demorar um pouco pra migrar para o lost, dei uma olhadinha por aqui, achei bem legal, porém não estou MUITO confortável com flebox e calc() (_eu sei que deveria está_).

**Comment by diegodap** [comment]
Estou testando o Lost através do Kratos do @LFeh , por enquanto não tive problemas, e para trabalhar com o básico ele é tão simples quanto o Jeet, só a documentação que realmente é bem ruim.

**Comment by kandebonfim** [comment]
Old thread... Mas só pra constar que o [jeet está de volta](http://jeet.gs/). 👍

**Comment by felipefialho** [comment]
Eita, mas e o Lost? 

Será que vão implementar as melhorias que o Lost trouxe?

**Comment by kandebonfim** [comment]
@LFeh não conhecia o Lost! Achei beeem irado!
